src/Components/Cart.js

```diff
@@ -0,0 +1,30 @@
+import firebase from '../firebase';
+import { getDatabase, ref, remove } from 'firebase/database';
+
+const Cart = (props) => {
```

Comment:
Super minor, but destructuring the props object would save a little bit of typing. Not a very big deal either way though.